src/lib_ccx/ccx_demuxer_mxf.c

```diff
@@ -307,7 +326,7 @@ static int mxf_read_cdp_data(struct ccx_demuxer *demux, int size, struct demuxer
 	cc_count = buffered_get_byte(demux) & 0x1F;
 	len++;
 	// -4 for cdp footer length
-	if ( (cc_count * 3) != (size - len - 4))
+	if ( (cc_count * 3) > (size - len - 4))
```

Comment:
Some files include extra padding, so this skips the warning when there's extra data, as opposed to missing data.